tests/test_openapi_tags.py

```diff
@@ -86,6 +87,8 @@ def test_auto_tag_from_blueprint(app, client):
     assert {'name': 'Foo'} in rv.json['tags']
 
 
+@pytest.mark.skipif(flask.__version__ < '2.0.1',
+                    reason='Depends on new behaviour introduced in Flask 2.0.1')
```

Comment:
Needed to add these skips as well if we're going to test with actual flask 2.0.0